dotnet/src/Connectors/Connectors.Amazon/Bedrock/Core/Models/CohereEmbed/EmbedResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Connectors.Amazon.Core;
+
+/// <summary>
+/// The Amazon Titan Text response object when deserialized from Invoke Model call.
+/// </summary>
+internal sealed class EmbedResponse
+{
+    /// <summary>
+    /// The number of tokens in the prompt.
+    /// </summary>
+    [JsonPropertyName("inputTextTokenCount")]
+    public int InputTextTokenCount { get; set; }
+
+    /// <summary>
+    /// The provided input text strings for text embedding response.
+    /// </summary>
+    [JsonPropertyName("texts")]
+    public IList<string>? Texts { get; set; }
+
+    /// <summary>
+    /// A list containing float arrays of the embeddings for each text string.
+    /// </summary>
+    [JsonPropertyName("embedding")]
```

Comment:
Small bug here. Should be in plural.

IT passed after this fix.

```suggestion
    [JsonPropertyName("embeddings")]
```